### ISSUE TITLE
Failed to build "master" branch (just now)

### ISSUE THREAD

**Issue by SchweinDeBurg** [doxygen/doxygen#7725]
Fedora 27, GCC 7.3
```
[ 69/280] Building CXX object src/CMakeFiles/doxycfg.dir/__/generated_src/configimpl.cpp.o
FAILED: src/CMakeFiles/doxycfg.dir/__/generated_src/configimpl.cpp.o 
/usr/bin/ccache  /usr/bin/g++  -DYY_BUF_SIZE=262144 -DYY_READ_BUF_SIZE=262144 -I../qtools -I../libmd5 -I../liblodepng -I../libmscgen -I../libversion -I../vhdlparser -I../src -Igenerated_src -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -m64 -march=native -mtune=native -fasynchronous-unwind-tables   -w -I/usr/include -O2 -g -DNDEBUG -fvisibility=hidden -fvisibility-inlines-hidden   -DLLVM_BUILD_GLOBAL_ISEL -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wpedantic -Wdouble-promotion -Wformat=2 -Wno-unused-parameter -Wno-sign-conversion -Wno-format-nonliteral -Wno-implicit-fallthrough -std=gnu++14 -MD -MT src/CMakeFiles/doxycfg.dir/__/generated_src/configimpl.cpp.o -MF src/CMakeFiles/doxycfg.dir/__/generated_src/configimpl.cpp.o.d -o src/CMakeFiles/doxycfg.dir/__/generated_src/configimpl.cpp.o -c generated_src/configimpl.cpp
In file included from /usr/include/c++/7/memory:80:0,
                 from /usr/include/c++/7/thread:39,
                 from configimpl.l:36:
/usr/include/c++/7/bits/unique_ptr.h: In instantiation of 'void std::default_delete<_Tp>::operator()(_Tp*) const [with _Tp = ArgumentList]':
/usr/include/c++/7/bits/unique_ptr.h:268:17:   required from 'std::unique_ptr<_Tp, _Dp>::~unique_ptr() [with _Tp = ArgumentList; _Dp = std::default_delete<ArgumentList>]'
../src/util.h:415:117:   required from here
/usr/include/c++/7/bits/unique_ptr.h:76:22: error: invalid application of 'sizeof' to incomplete type 'ArgumentList'
  static_assert(sizeof(_Tp)>0,
                      ^
ninja: build stopped: subcommand failed.
```

**Comment by albert-github** [comment]
Thanks, problem has been seen in the current master and reported (offline).
Please always mention  the sha id when reporting this kind of problems (here e.g. 9e2b6a2dac870b5daabde118c5327281e3f79ba2  causing the problem).

**Comment by albert-github** [comment]
An update has been applied (869f979013dab134ff5292fec21c7610516d514d ) and it looks that everything is compiling again.

**Comment by SchweinDeBurg** [comment]
Confirmed, many thanx!

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.19.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
